src/encoded/tests/data/inserts/higlass_view_config.json

```diff
@@ -49,39 +46,30 @@
                                     "trackBorderColor": "black"
                                 },
                                 "width": 1130,
-                                "name": "4DNFIWG6CQQA.beddb",
                                 "type": "horizontal-gene-annotations",
                                 "tilesetUid": "XGrAqZupQb2h5Pjiarf8PA",
-                                "header": "1\t2\t3\t4\t5\t6\t7\t8\t9\t10\t11\t12\t13\t14",
-                                "height": 55,
-                                "position": "top"
+                                "height": 55
                             },
                             {
                                 "server": "https://higlass.4dnucleome.org/api/v1",
                                 "uid": "top-chromosome-track",
-                                "thumbnail": null,
-                                "orientation": "1d-horizontal",
                                 "options": {
-                                    "plusStrandColor": "blue",
                                     "fontIsAligned": false,
                                     "color": "#777777",
                                     "labelColor": "#888",
                                     "mousePositionColor": "#999999",
                                     "fontSize": 12,
                                     "labelPosition": "hidden",
                                     "trackBorderWidth": 0,
-                                    "minusStrandColor": "red",
                                     "stroke": "#FFFFFF",
-                                    "showTooltip": false,                                    "showMousePosition": false,
+                                    "showTooltip": false,                                    
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6353740)